examples/SingleDatePickerWrapper.jsx

```diff
@@ -22,6 +22,10 @@ const propTypes = {
     'focused',
     'onFocusChange',
   ]),
+
```

Comment:
there shouldn't be two empty lines next to each other; let's collapse this to one